CMakeLists.txt

```diff
@@ -672,3 +677,8 @@ install(FILES ${${PROJECT_NAME}_INSTALL_QT_BEARER_LIBS} DESTINATION
 install(FILES ${${PROJECT_NAME}_INSTALL_QT_PLATFORMS_LIBS} DESTINATION          "platforms" OPTIONAL)
 install(FILES ${${PROJECT_NAME}_INSTALL_QT_PLAYLISTFORAMTS_LIBS} DESTINATION    "playlistformats" OPTIONAL)
 install(FILES ${${PROJECT_NAME}_INSTALL_QT_STYLES_LIBS} DESTINATION             "styles" OPTIONAL)
+
```

Comment:
Better solution make this OPTIONAL instead of putting an if around it and move it back to the original position --> Line 661.

On the other hand this should work if you would execute the custom build target Commander_Wars_translations before building, but having the translation files is optional so making them optional is fine.